packages/typescript-checker/run.js

```diff
@@ -0,0 +1,32 @@
+const { TypescriptChecker } = require('./dist/src/TypescriptChecker');
+const { resolve } = require('path');
+const { readFileSync } = require('fs');
+const { default: MutantStatus } = require('@stryker-mutator/api/src/report/MutantStatus');
+
+async function run() {
```

Comment:
Did you want to commit this file?

Reply:
Nope. Created this file before I migrated to integration tests. Good find!